libs/types/src/smartcard_auth.ts

```diff
@@ -146,7 +157,12 @@ export interface LoggedOutAuth {
     | 'invalid_user_on_card'
     | 'user_role_not_allowed'
     | 'machine_not_configured'
-    | 'pollworker_election_hash_mismatch';
+    | 'pollworker_wrong_election'
+    | 'voter_wrong_election'
```

Comment:
Once we finish the VVSG2 auth project, `'admin_wrong_election'` will also be a possible value, since super admins will be programming admin cards for specific elections

I can always add this in when it becomes relevant! But just wanted to note

Reply:
Good to know! Hopefully it's straightforward to add when needed